brave/browser-laptop
# Fixes backup keys export/import; improves print flow

Resolves #13274
Resolves #7512
Resolves #7511
Resolves #11419

## Submitter Checklist:


## Test Plan:
Plan a (#13274)

1. Enable payments
2. Go to advance settings
3. Export backup keys

Plan b (#13274)

1. Enable payments
2. Go to advance settings
3. Import backup keys

Plan c (#7512)

1. Enable payments
2. Go to advance settings
3. Go to backup
4. Click print
5. Make sure that new tab is opened and print dialog is shown

Plan d (#7511)

1. Enable payments
2. Go to advance settings
3. Export backup keys
4. Make sure that new tab is not opened

Plan e (#11419)

1. Enable payments
2. Go to advance settings
3. Export backup keys
4. Go to brave profile folder and make sure that `brave_wallet_recovery.txt` is not there

## Reviewer Checklist:


Tests
